### ISSUE TITLE
Status after approval

### ISSUE THREAD

**Issue by JoaoFerreiraEla** [mageplaza/magento-2-customer-approval#24]
Hello,

I'm running the 4.0.0 version of the module with Magento 2.4.1.

When I approve a customer from the backend, it goes to the status "new" instead of approved. This issue only started after I upgraded my Magento version from 2.3.5 to 2.4.1. Can anyone help?

Thanks in advance.

**Comment by HuyPhuc98** [comment]
Hi,

To fix your problem, please go to Database and delete the line Mageplaza_CustomerApproval in the setup_module table, and delete the is_approved line in the eav_attribute table || https://prnt.sc/wctuks || https://prnt.sc/wctv24
Then run the upgrade and deploy commands to reinstall the Customer Approval module.

Thank you!

**Comment by JoaoFerreiraEla** [comment]
It didn't solve the issue, a still get "new" on the instead of approved. Any other ideas?

**Comment by JoaoFerreiraEla** [comment]
Ok, so I found the reason why this happened to me.... Sorry to bother you guys.

So from what I saw on the modules code versions, the default value of the is_approved field was changed in August from "approved" to "new".

What is happening is, when I update a customer using Magento REST API, since I'm not passing the approved field it changes to the default value witch is "new".

Sorry for wasting your time and thank you for this great extension 👍
